TokenAdministrationApi.Tests/V1/E2ETests/PostTokenIntegrationTests.cs

```diff
@@ -10,23 +10,24 @@
 using TokenAdministrationApi.Tests.V1.Helper;
 using TokenAdministrationApi.V1.Boundary.Requests;
 using TokenAdministrationApi.V1.Boundary.Response;
+using TokenAdministrationApi.V1.Infrastructure;
 
 namespace TokenAdministrationApi.Tests.V1.E2ETests
 {
-    public class PostTokenIntegrationTests : IntegrationTestsPost<Startup>
+    public class PostTokenIntegrationTests : IntegrationTests<Startup>
     {
         private readonly Faker _faker = new Faker();
-
         [Test]
         public async Task CanGenerateAnAuthTokenAsync()
         {
+            var (apiLookup, apiEndpointLookup, consumerTypeLookup) = AddLookupsToDatabase();
```

Comment:
Is this a way to assign a single value to multiple variables?
e.g. Does this assign the result of `AddLookupsToDatabase()` to the variables: `apiLookup`, `apiEndpointLookup` & `consumerTypeLookup`?

Reply:
Yeah so the method returns a tuple, which contains multiple values and then this assignment will deconstruct the tuple